src/python/gem5/utils/multisim/multisim.py

```diff
@@ -51,13 +51,17 @@
 
 import importlib
 import multiprocessing
+import signal
+import time
 from pathlib import Path
+from threading import Lock
```

Comment:
This should be
```suggestion
from multiprocessing import Lock
```

The threading lock is not compatible with multiprocessing.